cxbox-core/src/main/java/org/cxbox/core/dao/impl/MetadataUtils.java

```diff
@@ -415,4 +462,112 @@ private boolean isElementCollectionField(Root<?> root, String fieldName) {
+//			return (root, query, cb) -> {
+//				Expression<String> timeExpr = cb.function("TO_CHAR", String.class, field, cb.literal("HH24:MI:SS"));
+//				return cb.greaterThan(timeExpr, time.format(DateTimeFormatter.ofPattern("HH:mm:ss")));
+//			};
+//		}
+//		return (root, query, cb) -> cb.greaterThan(field.as(LocalTime.class), time);
+//	}/**
+//	 * Filters entity column by time fraction greaterThen (LocalDateTime TODO>>other supported types??? Types used in projects? Which types from projects are not supported?)
+//	 * {@code dialect (Oracle/PostgreSQL)} - entityManager.getEntityManagerFactory().getProperties().get("hibernate.dialect").toString();
+//	 * <br>
+//	 * <h6>dialect = PostgreSQL</h6>
+//	 * Column in DB: TIMESTAMP/ TODO>>other supported types??? Types used in projects? Which types from projects are not supported?
+//	 * <br>
+//	 * Actual SQL:
+//	 * <br>
+//	 * {@code Query:["select a1_0.id,a1_0.commend,a1_0.created_date from apple a1_0 where cast(a1_0.created_date as time)>?"]}
+//	 * so always add functional index CREATE INDEX idx_apple_created_date_time ON apple ((created_date::time));
+//	 * <br>
+//	 * <h6>dialect = Oracle</h6>
+//	 * Column in DB: TIMESTAMP/ TODO>>other supported types??? Types used in projects? Which types from projects are not supported?
+//	 * <br>
+//	 * Actual SQL:
+//	 * <br>
+//	 * {@code Query:["select a1_0.id,a1_0.commend,a1_0.created_date from apple a1_0 where to_char(a1_0.created_date,'HH24:MI:SS')>?"]}
+//	 * so always add functional index ?????????;
+//	 * <br>
+//	 */
+//	static Specification byTimeAfter(@NonNull LocalTime time, @Nullable String dialect, @NonNull Path field) {
+//		if (dialect.contains("Oracle")) {
+//			return (root, query, cb) -> {
+//				Expression<String> timeExpr = cb.function("TO_CHAR", String.class, field, cb.literal("HH24:MI:SS"));
+//				return cb.greaterThan(timeExpr, time.format(DateTimeFormatter.ofPattern("HH:mm:ss")));
+//			};
+//		}
+//		return (root, query, cb) -> cb.greaterThan(field.as(LocalTime.class), time);
+//	}
+
+	/**
+	 * Sorts entity column by time fraction (LocalDateTime TODO>>other supported types??? Types used in projects? Which types from projects are not supported?)
+	 * <br>
+	 * {@code dialect (Oracle/PostgreSQL)} - entityManager.getEntityManagerFactory().getProperties().get("hibernate.dialect").toString();
+	 * <br>
+	 * <h6>dialect = PostgreSQL</h6>
+	 * Column in DB: TIMESTAMP/ TODO>>other supported types??? Types used in projects? Which types from projects are not supported?
+	 * <br>
+	 * Actual SQL:
+	 * <br>
+	 * {@code Query:["select a1_0.id,a1_0.commend,a1_0.created_date from apple a1_0 order by cast(a1_0.created_date as time) asc"]}
+	 * so always add functional index CREATE INDEX idx_apple_created_date_time ON apple ((created_date::time));
+	 * <br>
+	 * <h6>dialect = Oracle</h6>
+	 * Column in DB: TIMESTAMP/ TODO>>other supported types??? Types used in projects? Which types from projects are not supported?
```

Comment:
TODO>>other supported types??? what is that? you need to feel this with valid doc